examples/text_classification/ios/TextClassifier/ViewController.swift

```diff
@@ -0,0 +1,125 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import MediaPipeTasksText
+
+class ViewController: UIViewController {
+
+    // IBOutlet
+    @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var choseModelButton: UIButton!
+    @IBOutlet weak var inputTextView: UITextView!
+    @IBOutlet var titleView: UIView!
+    @IBOutlet weak var inferenceTimeLabel: UILabel!
+    @IBOutlet weak var settingViewHeightLayoutConstraint: NSLayoutConstraint!
+
+    // variable
+    var defaultModel = Model.mobileBert
+    var textclassitifier: TextClassifierHelper!
+    var datas: [ResultCategory] = []
```

Comment:
Data can be singular (data instead of datas)

Reply:
Actually let's call this 'categories' since that's what we're storing in the array.